### DISCUSSION TITLE
payment.log in magento/module-payment

### DISCUSSION THREAD

**Discussion by vish1plunker** [markshust/docker-magento#624] [Q&A] [answered]
I am not able to use payment.log in docker anyone know how to make that working? magento/module-payment has custom payment.log that gets enabled and can be used via custom plugin logging but somehow its not working with this docker

**Comment by rangerz** [comment, marked as answer]
The custom log (`/var/log`) is stored in docker container.

Try this command:
```
bin/cli tail -f ./var/log/payment.log
```

or uncomment this line and restart docker (`bin/restart`)
https://github.com/markshust/docker-magento/blob/4331507f9e252961bc98fc382602b0f8925ab4e4/compose/docker-compose.dev.yml#L16
The log will write into your local folder

**Reply by vish1plunker** [reply]
Perfect ! worked with commenting line in .uml file
